crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -979,10 +998,16 @@ impl CumulocityConverter {
         channel: Channel,
         message: &MqttMessage,
     ) -> Result<Vec<MqttMessage>, ConversionError> {
-        let mut registration_messages: Vec<MqttMessage> = vec![];
+        let mut converted_messages: Vec<MqttMessage> = vec![];
```

Comment:
All changes under this function `try_convert_te_topics` will be changed after merging https://github.com/thin-edge/thin-edge.io/pull/2884. Ignore the changes in this function for now.